include/mxnet/imperative.h

```diff
@@ -35,6 +35,14 @@
 #include "./ndarray.h"
 
 namespace mxnet {
+
+constexpr char OPT_CONSTRAINT_ATTR[] = "__opt_constraint__";
+enum class OptConstraint : int {
+  None       = 0,
+  DisableAMP = 1 << 0
+  // DisableQuantization = 1 << 1
+};
+
```

Comment:
I was wondering if there is a better place for this. @bgawrych @anko-intel @bartekkuncer